readme.md

```diff
@@ -146,6 +166,8 @@ mkvirtualenv wagtailbakerydemo --python=python3.12
 python --version
 ```
 
+# Essential Setup
+
```

Comment:
```suggestion
```

This part of the setup is only relevant for virtual environment users so shouldn’t have its own section.